test/e2e/storage/csi_volumes.go

```diff
@@ -367,57 +385,96 @@ func initCSIgcePD(f *framework.Framework, config framework.VolumeTestConfig) csi
 	framework.SkipUnlessSecretExistsAfterWait(cs, "cloud-sa", config.Namespace, 3*time.Minute)
 
 	return &gcePDCSIDriver{
-		nodeClusterRoles: []string{
-			csiDriverRegistrarClusterRoleName,
-		},
-		controllerClusterRoles: []string{
-			csiExternalAttacherClusterRoleName,
-			csiExternalProvisionerClusterRoleName,
-		},
 		f:      f,
 		config: config,
 	}
 }
 
-func (g *gcePDCSIDriver) createStorageClassTest(node v1.Node) storageClassTest {
+func (g *gcePDCSIDriver) createStorageClassTest() storageClassTest {
 	return storageClassTest{
 		name:         "com.google.csi.gcepd",
-		provisioner:  "com.google.csi.gcepd",
+		provisioner:  "com.google.csi.gcepd-" + g.f.UniqueName,
 		parameters:   map[string]string{"type": "pd-standard"},
 		claimSize:    "5Gi",
 		expectedSize: "5Gi",
-		nodeName:     node.Name,
+		nodeName:     g.config.ServerNodeName,
 	}
 }
 
 func (g *gcePDCSIDriver) createCSIDriver() {
 	By("deploying gce-pd driver")
-	f := g.f
-	cs := f.ClientSet
-	config := g.config
-	g.controllerServiceAccount = csiServiceAccount(cs, config, "gce-controller", false /* teardown */)
-	g.nodeServiceAccount = csiServiceAccount(cs, config, "gce-node", false /* teardown */)
-	csiClusterRoleBindings(cs, config, false /* teardown */, g.controllerServiceAccount, g.controllerClusterRoles)
-	csiClusterRoleBindings(cs, config, false /* teardown */, g.nodeServiceAccount, g.nodeClusterRoles)
-	utils.PrivilegedTestPSPClusterRoleBinding(cs, config.Namespace, false, /* teardown */
-		[]string{g.controllerServiceAccount.Name, g.nodeServiceAccount.Name})
-	role := csiControllerRole(cs, config, false)
-	csiControllerRoleBinding(cs, config, false, role, g.controllerServiceAccount)
-	deployGCEPDCSIDriver(cs, config, false /* teardown */, f, g.nodeServiceAccount, g.controllerServiceAccount)
+	cleanup, err := g.f.CreateFromManifest(func(object interface{}) {
```

Comment:
fyi, I am planning to launch the gce pd csi driver outside of the e2e test, and disable gce pd csi tests from kubernetes/kubernetes, and run it from kubernetes-sigs/gcp-compute-persistent-disk-csi-driver.  So don't spend too much effort changing gce pd.